src/js/components/projects/listItem.tsx

```diff
@@ -1,75 +1,79 @@
 
-const ProjectListItem = ({ project }: { project: Project }) => (
-  <div
-    className="metecho-project-item
-      slds-p-around_small
-      slds-size_1-of-1
-      slds-medium-size_1-of-2
-      slds-large-size_1-of-3"
-  >
-    <Link
-      to={routes.project_detail(project.slug)}
-      className="slds-box
-        slds-box_link
-        slds-box_x-small
-        slds-theme_default
-        slds-media
-        container-fill-space"
+const ProjectListItem = ({ project }: { project: Project }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div
+      className="metecho-project-item
+        slds-p-around_small
+        slds-size_1-of-1
+        slds-medium-size_1-of-2
+        slds-large-size_1-of-3"
     >
-      <div className="slds-media__figure slds-m-left_xx-small">
-        {project.repo_image_url ? (
-          <div>
-            <img
-              src={project.repo_image_url}
-              alt={t('social image for {{title}}', {
-                title: project.name,
-              })}
-            />
-          </div>
-        ) : (
-          <Avatar
-            variant="entity"
-            size="large"
-            label={project.name}
-            title={project.name}
-          />
-        )}
-      </div>
-      <div
-        className="slds-media__body
-          slds-border_left
-          slds-p-around_small
-          slds-grid
-          slds-grid_vertical
-          content-fill-height"
+      <Link
+        to={routes.project_detail(project.slug)}
+        className="slds-box
+          slds-box_link
+          slds-box_x-small
+          slds-theme_default
+          slds-media
+          container-fill-space"
       >
-        <h2
-          className="slds-truncate slds-text-heading_small"
-          title={project.name}
-        >
-          {project.name}
-        </h2>
-        <div className="slds-m-top_small">
-          {project.description_rendered ? (
-            <div
-              className="truncate-children"
-              // This description is pre-cleaned by the API
-              dangerouslySetInnerHTML={{
-                __html: project.description_rendered,
-              }}
+        <div className="slds-media__figure slds-m-left_xx-small">
+          {project.repo_image_url ? (
+            <div>
+              <img
+                src={project.repo_image_url}
+                alt={t('social image for {{title}}', {
+                  title: project.name,
+                })}
+              />
+            </div>
+          ) : (
+            <Avatar
+              variant="entity"
+              size="large"
+              label={project.name}
+              title={project.name}
             />
-          ) : null}
+          )}
+        </div>
+        <div
+          className="slds-media__body
+            slds-border_left
+            slds-p-around_small
+            slds-grid
+            slds-grid_vertical
+            content-fill-height"
+        >
+          <h2
```

Comment:
Speaking of header ordering, it appears the list component is using an H2 tag, but then the listItems are also using an H2. The heading order here should be taken into consideration.

Reply:
Good catch! Fixed in https://github.com/SFDO-Tooling/Metecho/pull/2019/commits/8b50b4fc5c793962be86b31a7706446e63cc8a49